### ISSUE TITLE
step 1 baseline_280M loss large

### ISSUE THREAD

**Issue by gawei1995** [sangmichaelxie/doremi#1]
280M baseline model loss is hovering around 5, with all training hyperparameters set default values.
 The preprocess file sampler is set to 10w
![image](https://github.com/sangmichaelxie/doremi/assets/56674565/a0d56065-681b-4127-a679-8245accd0817)

![image](https://github.com/sangmichaelxie/doremi/assets/56674565/3b104c1e-30fe-42d7-a123-c853b975eb5d)

**Comment by gawei1995** [comment]
i find the bug trainer parameters is 40M  , not 380M ,but why ????

![image](https://github.com/sangmichaelxie/doremi/assets/56674565/3ae0046a-287c-46f2-9969-b214d8d2c5fc)


![image](https://github.com/sangmichaelxie/doremi/assets/56674565/b1dc242b-4593-4857-b61d-0aea2af23dc6)

**Comment by sangmichaelxie** [comment]
The loss value around 5 is expected, and things should be working normally for you. The number of trainable parameters being smaller is due to sharding across the GPUs. If you try with 1 GPU it should say the full number.

**Comment by gawei1995** [comment]
> The loss value around 5 is expected, and things should be working normally for you. The number of trainable parameters being smaller is due to sharding across the GPUs. If you try with 1 GPU it should say the full number.

thank you for  replying to me，but when i use the gpt2 official version not ur gpt2fast version, the loss reached 2.4. so i think it's a model architecture issue

**Comment by gawei1995** [comment]
> The loss value around 5 is expected, and things should be working normally for you. The number of trainable parameters being smaller is due to sharding across the GPUs. If you try with 1 GPU it should say the full number.

step1  loss when use gpt2  official  version

![image](https://github.com/sangmichaelxie/doremi/assets/56674565/a1f38c5c-6451-411b-91c6-3c25372f7e7f)

**Comment by sangmichaelxie** [comment]
Actually, you're right, thanks for pointing this out. I've replaced the model in the current HEAD; you'll need to run `bash scripts/setup_flash.sh` before running it again. In prelim tests the loss goes much lower than before.
